Documentation/security/network/encryption-ipsec.rst

```diff
@@ -10,6 +10,13 @@
 IPsec Transparent Encryption
 ****************************
 
+.. warning::
+
+   IPsec does not work when using :ref:`kube-proxy replacement <kubeproxy-free>`.
+   Be aware that other features may require a kube-proxy free environment in
+   which case they are mutual exclusive.
```

Comment:
Could we instead add it to the limitations at the end of the page? I don't think it requires a bigger mention than other limitations.

Reply:
Done 👍